src/types.h

```diff
@@ -329,6 +329,17 @@ inline Score operator/(Score s, int i) {
   return make_score(mg_value(s) / i, eg_value(s) / i);
 }
 
+/// Multiplication of a Score by an integer. We check for overflow in debug mode.
+inline Score operator*(Score s, int i) {
+  Score result = Score(int(s) * i);
+
+  assert( eg_value(result) == (i * eg_value(s)));
+  assert( mg_value(result) == (i * mg_value(s)));
+  assert( (i == 0) || (result / i) == s );
```

Comment:
@mcostalba: done in the new pull request https://github.com/official-stockfish/Stockfish/pull/969